api/index.js

```diff
@@ -321,6 +321,13 @@ app.use('/', (req, res) => {
   })
 });
 
+app.use((err, req, res, next) => {
```

Comment:
Will it be possible to identify an API endpoint by the log?

Reply:
Yes, here is the log example:
`Error: balances.totalIssuance() does not take any arguments, 1 found
    at assert (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/@polkadot/util/assert.cjs:34:11)
    at extractStorageArgs (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/@polkadot/api/util/validate.cjs:49:20)
    at overrideNoSub (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/@polkadot/api/base/Decorate.cjs:419:99)
    at /media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/@polkadot/api/promise/index.cjs:73:26
    at new Promise (<anonymous>)
    at decorateCall (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/@polkadot/api/promise/index.cjs:69:10)
    at Object.totalIssuance (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/@polkadot/api/promise/index.cjs:101:73)
    at Object.getTotalSupply (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/api/src/services/cereNetworkService.js:93:45)
    at getTotalSupplyInternal (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/api/src/services/cereTokensService.js:10:29)
    at getTotalSupply (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/api/src/services/cereTokensService.js:16:33)
    at Layer.handle [as handle_request] (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/express/lib/router/route.js:137:13)
    at makeResponseCacheable (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/apicache/src/apicache.js:242:5)
    at cache (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/apicache/src/apicache.js:689:16)
    at Layer.handle [as handle_request] (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/express/lib/router/layer.js:95:5)
    at next (/media/dmitriy/888e219d-113d-48ba-a562-be15bc3e5d342/Projects/cere/cerestats/node_modules/express/lib/router/route.js:137:13)
GET /api/v1/tokens/total-supply 500 7.740 ms - 37`

Reply:
fixed